contracts/interchain-token-service/src/interface.rs

```diff
@@ -0,0 +1,58 @@
+use axelar_soroban_std::types::Token;
+use soroban_sdk::{contractclient, Address, Bytes, Env, String};
+
+use crate::error::ContractError;
+
+#[contractclient(name = "InterchainTokenServiceClient")]
+pub trait InterchainTokenServiceInterface {
+    fn owner(env: &Env) -> Address;
+
+    fn transfer_ownership(env: Env, new_owner: Address);
+
+    fn trusted_address(env: &Env, chain: String) -> Option<String>;
+
+    fn set_trusted_address(env: Env, chain: String, address: String) -> Result<(), ContractError>;
+
+    fn remove_trusted_address(env: Env, chain: String) -> Result<(), ContractError>;
+
+    fn execute_message(
```

Comment:
this is the internal _execute method, exposing it as an entrypoint is a security issue since it bypasses validation

Reply:
make sense, updated